From 74f6cdcefac9a4f6cd5670af58893267c50b6c09 Mon Sep 17 00:00:00 2001
From: Zenon Mousmoulas <zmousm@noc.grnet.gr>
Date: Fri, 16 Mar 2018 10:27:15 +0200
Subject: [PATCH] Simplify spooled value regex

Validating the value is irrelevant in SpoolReader, so the regular
expression can be very simple.
---
 node/lib/Munin/Node/SpoolReader.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/node/lib/Munin/Node/SpoolReader.pm b/node/lib/Munin/Node/SpoolReader.pm
index 5add1207..e3de5f73 100644
--- a/node/lib/Munin/Node/SpoolReader.pm
+++ b/node/lib/Munin/Node/SpoolReader.pm
@@ -148,7 +148,7 @@ sub _cat_multigraph_file
                 next;
             }
 
-            if (m/^(\w+)\.value\s+(?:N:)?(-?[0-9.]+|U)$/) {
+            if (m/^(\w+)\.value\s+(?:N:)?(.+)$/) {
                 $_ = "$1.value $epoch:$2";
             }
 
-- 
2.16.2

